pkg/virt-operator/resource/generate/components/prometheus.go

```diff
@@ -552,14 +559,24 @@ func NewPrometheusRuleSpec(ns string, workloadUpdatesEnabled bool) *v1.Prometheu
 							"runbook_url": runbookUrlBasePath + "KubeVirtComponentExceedsRequestedCPU",
 						},
 						Labels: map[string]string{
-							"severity": "warning",
+							severityAlertLabelKey: "warning",
 						},
 					},
 				},
 			},
 		},
 	}
 
+	for _, group := range ruleSpec.Groups {
```

Comment:
nit: can do this after the `if workloadUpdatesEnabled {` below so that the labels are "auto-added" to the OutdatedVirtualMachineInstanceWorkloads alert too.

Reply:
Done, thanks :)